vendor/priors/gwt.js

```diff
@@ -12,6 +12,15 @@ function convertToByteArray(target) {
     return target;
 }
 
+function propsToFragment(props) {
```

Comment:
I'm not sure i like the word fragment also both these functions are just 1 liners. Does it make sense to even have functions rather then call inline?

Reply:
Fragment is standard and I'm trying to avoid confusion with cryptographic hashes: https://en.wikipedia.org/wiki/Fragment_identifier

These functions are called from multiple components, and I don't want to risk making one out of sync with the other.

Reply:
That was new to me! all ok.